src/components/SingleArticle/SingleArticle.module.scss

```diff
@@ -4,6 +4,14 @@
   margin-top: units(6);
   margin-bottom: units(4);
 
+  max-width: 948px;
+
+  .hero {
+    width: 100%;
+    max-height: 340px;
+    object-fit: cover;
+  }
+
   h2 {
     margin-bottom: 0;
```

Comment:
We need to add `16px `under the `h2` since we moved the tag below it. In yours, the heading is almost touching the tag. See figma screenshot below for reference.

<img width="298" alt="image" src="https://user-images.githubusercontent.com/59394696/206798933-93b527b7-19af-41b1-936f-1620145f8903.png">